common/utilities/string/wrap.cpp

```diff
@@ -0,0 +1,130 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include <string>
+#include <vector>
+#include <sstream>
+#include "wrap.h"
+#include "split.h"
+#include "../third-party/imgui/imgui.h"
+
+namespace utilities {
+namespace string {
+
+std::string wrap_paragraph( const std::string & paragraph, int wrap_pixels_width )
+{
+    float space_width = ImGui::CalcTextSize( " " ).x;  // Calculate space width in pixels
+    std::string wrapped_line;                          // The line that is wrapped in this iteration
+    std::string wrapped_paragraph;                     // The output wrapped paragraph
+    auto remaining_paragraph = paragraph;              // Holds the remaining unwrapped part of the input paragraph
+
+    // Handle a case when the paragraph starts with spaces
+    if( remaining_paragraph[0] == ' ' )
```

Comment:
I think maybe separate into `find_first_word( remaining_paragraph )` since it repeats again later on...
But in any case:
I don't see the need for this if. The `find_first_not_of` is always good, and will return 0.

Reply:
Correct , Done